bin/mv.js

```diff
@@ -107,17 +108,25 @@ function copy(profile, webtask, target) {
     claims.jtn = target.name || claims.jtn;
     claims.ten = target.container || claims.ten;
 
+    var pendingCreate;
     if (target.profile) {
-        create = loadProfile(target.profile)
+        pendingCreate = loadProfile(target.profile)
             .then(function(profile) {
+                targetProfile = profile;
                 claims.ten = target.container || profile.container || claims.ten;
-                return profile.createRaw(claims);
+                return targetProfile.createRaw(claims);
             });
     } else {
-        create = profile.createRaw(claims);
+        pendingCreate = targetProfile.createRaw(claims);
     }
 
-    return create
+    return pendingCreate
+        .then(function() {
+            target.profile = targetProfile;
+            target.name = claims.jtn;
```

Comment:
Perhaps we should make sure that this command does an early exit if the source webtask is not a _named webtask_ (does not have `jtn` claim).